Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -232,6 +232,7 @@
         reagents:
         - ReagentId: Ephedrine
           Quantity: 15
+  - type: TrashOnEmpty
```

Comment:
with the changes above these can be removed and put on the base `Syringe` instead